compiler/rustc_resolve/src/late/lifetimes.rs

```diff
@@ -2013,9 +2013,6 @@ impl<'a, 'tcx> LifetimeContext<'a, 'tcx> {
             })
             .collect();
 
-        // ensure that we issue lints in a repeatable order
-        def_ids.sort_by_cached_key(|&def_id| self.tcx.def_path_hash(def_id));
```

Comment:
@cjgillot is this fine?

Reply:
> @cjgillot is this fine?

@bjorn https://github.com/rust-lang/rust/pull/89836#discussion_r738610531